requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+pytest
```

Comment:
We can just place this into the requirements.txt.

The dependencies listed in `setuptools.py` are for the end-user and the ones in the requirements.txt for local development.

Reply:
OK, I'll remove this file!